.github/workflows/continuous-integration.yml

```diff
@@ -75,7 +123,7 @@ jobs:
         # -q: quiet
         unzip -q opensim-core-mac.zip -d ~
     - name: Checkout opensim-gui
-      uses: actions/checkout@v1
+      uses: actions/checkout@v2
```

Comment:
Can the location of the step that checks out the GUI be consistent between all platforms?